conftest.py

```diff
@@ -13,13 +15,18 @@ def pytest_generate_tests(metafunc):
 
 
 def pytest_addoption(parser):
-    parser.addoption('--provider', default='tinkergraph',
-                     choices=('tinkergraph', 'dse',))
```

Comment:
I don't see any style improvement here, again, what motivates this revision?

Reply:
`yapf` default - no personal opinion